src/sql/workbench/services/contextualization/common/serverContextualizationService.ts

```diff
@@ -65,18 +69,28 @@ export class ServerContextualizationService extends Disposable implements IServe
 		// Don't need to take any actions if contextualization is not enabled and can return
 		const isContextualizationNeeded = await this.isContextualizationNeeded();
 		if (!isContextualizationNeeded) {
+			this._logService.info('Contextualization is not needed because the GitHub Copilot extension is not installed and/or contextualization is disabled.');
 			return;
 		}
 
 		const getServerContextualizationResult = await this.getServerContextualization(uri);
 		if (getServerContextualizationResult.context) {
+			this._logService.info('Server contextualization was previously generated for the URI connection, so sending that to Copilot for context.');
```

Comment:
is it also useful to put the uri for these messages? If there are several connections with their context getting generated, then that would help figure out which one is having problems